File: OPAL/br/src/main/scala/org/opalj/br/instructions/ISUB.scala
/* BSD 2-Clause License - see OPAL/LICENSE for details. */
package org.opalj
package br
package instructions

/**
 * Subtract int.
 *
 * @author Michael Eichberg
 */
case object ISUB extends SubtractInstruction {

    final val opcode = 100

    final val mnemonic = "isub"

    final val computationalType = ComputationalTypeInt

}

File: OPAL/bi/src/test/fixtures-java/projects/java8methodresolution/ISub.java
/* BSD 2-Clause License - see OPAL/LICENSE for details. */
package java8methodresolution;

import static java.lang.System.out;

public interface ISub extends ISuper, ISuperAlt{

	
	// we have to override!
	@Override
	default int compute(int a, int b) {
		out.println("ISub.compute");
		return ISuperAlt.super.compute(a,ISuper.super.compute(a, b));
	}
	
}
